platform/ios/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ## master
 
 * Fixed a crash caused by incorrect `MGLMapViewImpl` renderable size. ([#14810](https://github.com/mapbox/mapbox-gl-native/pull/14810))
+* Fixed an issue that caused the map to ignore the remote fonts for CJK Kanji glyphs. Added ability to specify an array of fonts for fallbacks for local ideographic font family.([#14862](https://github.com/mapbox/mapbox-gl-native/pull/14862))
```

Comment:
This changelog entry seems to say that we now use remote fonts for kanji, which isn’t the case.

```suggestion
* The `MGLIdeographicFontFamilyName` Info.plist key can now be set to an array of font family names to ensure correct font fallback behavior. It can also be set to a Boolean value of `NO` to force the SDK to typeset CJK characters in a remote font specified by `MGLSymbolStyleLayer.textFontNames`. ([#14862](https://github.com/mapbox/mapbox-gl-native/pull/14862))
```

Also, please add the same entry to the macOS changelog.